modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -607,6 +607,58 @@ public void testAddObjectEntry() throws Exception {
 			"{\"le\": \"2020-01-02\", \"ge\": \"2020-01-02\"}");
 
 		_assertCountAggregationObjectFieldValue(1, parentObjectEntry1);
+
+		ObjectDefinition objectDefinitionPickList = _createObjectDefinition(
+			Arrays.asList(
+				new PicklistObjectFieldBuilder(
+				).indexed(
+					true
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).listTypeDefinitionId(
+					_listTypeDefinition.getListTypeDefinitionId()
+				).name(
+					"picklistObjectFieldName"
+				).objectFieldSettings(
+					Collections.emptyList()
+				).build()));
+
+		ListTypeEntry listTypeEntry =
+			_listTypeEntryLocalService.addListTypeEntry(
+				null, _adminUser.getUserId(),
+				_listTypeDefinition.getListTypeDefinitionId(),
+				RandomTestUtil.randomString(),
+				Collections.singletonMap(
+					LocaleUtil.US, RandomTestUtil.randomString()));
+
+		ListEntry pickListListEntry = new ListEntry();
+
+		pickListListEntry.setKey(listTypeEntry.getKey());
+		pickListListEntry.setName(listTypeEntry.getName(LocaleUtil.US));
+
+		ObjectEntry childObjectEntryListEntry = new ObjectEntry() {
```

Comment:
@henriquelimadotcom please rename it to `objectEntry`

Reply:
Done!